packages/polaris-viz/src/components/Grid/Grid.tsx

```diff
@@ -89,6 +105,13 @@ export function Grid(props: GridProps) {
     return new Set([group.id, ...(group.connectedGroups ?? [])]);
   };
 
+  useLayoutEffect(() => {
```

Comment:
I think we can get rid of this and use the `TOOLTIP_MIN_HEIGHT` constants after we rename `TOOLTIP_VERTICAL_OFFSET` into `TOOLTIP_MIN_HEIGHT`